Improve checkpoint-sync-url docs (#571)
Consensys/doc.teku: docs/get-started/checkpoint-start.md

```

The command uses the [`--checkpoint-sync-url`](../reference/cli/index.md#checkpoint-sync-url) option

> to download the finalized checkpoint state. Please note that when using this option, you must delete your existing beacon node database (e.g. `<data-path>/beacon/db`) so Teku can download the new finalized state that will be used for your node.

Reviewer: ```suggestion
to download the finalized checkpoint state. When using this option, you must delete your existing beacon node database, for example `<data-path>/beacon/db`, to enable Teku to download the new, finalized state for your node.
```
